tests/conftest.py

```diff
@@ -26,6 +27,116 @@ def database(request, pymongo_client):
     return database
 
 
+def batch(
```

Comment:
This one also. You create the fixture but don't use it. And if you plan to use it, create the variables on the fly using a list like this [13,18,21,X,Y]?